lib/signalfx/tracing/sfx_logger.rb

```diff
@@ -0,0 +1,38 @@
+require 'logger'
+
+module SignalFx
+  module Tracing
+    class SFxLogger < Logger
+
+      attr_accessor :log
+
+      LOG_LEVELS = {
+        'unknown' =>Logger::UNKNOWN,
+        'fatal' => Logger::FATAL,
+        'error' =>  Logger::ERROR,
+        'warn' => Logger::WARN,
+        'info' => Logger::INFO,
+        'debug' => Logger::DEBUG
+      }.freeze
+
+      def initialize (dirname = ENV['SIGNALFX_LOG_DIRECTORY'] || 'sfx_logs',
+                      filename =  ENV['SIGNALFX_LOG_FILE'] || 'tracing.log',
+                      duration = ENV['SIGNALFX_LOG_DURATION'])
```

Comment:
It may be better to have users specify a rotation count and max size.  At the least these should be sanitized (with a default?).